akka-stream-tests/src/test/scala/akka/stream/scaladsl/ActorRefBackpressureSinkSpec.scala

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.scaladsl
+
+import akka.actor.{ Actor, ActorRef, Props }
+import akka.stream.ActorMaterializer
+import akka.stream.testkit.Utils._
+import akka.stream.testkit._
+import akka.stream.testkit.scaladsl._
+
+object ActorRefBackpressureSinkSpec {
+  val initMessage = "start"
+  val completeMessage = "done"
+  val ackMessage = "ack"
+
+  class Fw(ref: ActorRef) extends Actor {
+    def receive = {
+      case msg if msg == initMessage ⇒
```

Comment:
you can either use capital letters (i.e. name it `InitMessage`) or surround by backticks